### ISSUE TITLE
[DOC] metric description is required

### ISSUE THREAD

**Issue by baolinCloud** [shirou/gopsutil#384]
for example,

# cpu
 %user  – Percentage of CPU utilization that occurred while executing at the user level (application). Note that this field includes time spent running virtual processors.

%nice  – Percentage of CPU utilization that occurred while executing at the user level with nice priority.

**Comment by shirou** [comment]
Indeed! However, I am not so familiar with English, documentation PR is always welcome!
